python/paddle/fluid/contrib/slim/quantization/mkldnn_post_training_strategy.py

```diff
@@ -0,0 +1,116 @@
+# Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import logging
+import six
+import numpy as np
+from .... import core
+from ..core.strategy import Strategy
+
+__all__ = ['MKLDNNPostTrainingQuantStrategy']
+
+logging.basicConfig(format='%(asctime)s-%(levelname)s: %(message)s')
+_logger = logging.getLogger(__name__)
+_logger.setLevel(logging.INFO)
+
+
+class MKLDNNPostTrainingQuantStrategy(Strategy):
+    """
+    The strategy for Post Training quantization strategy.
+    """
+
+    def __init__(self,
+                 int8_model_save_path=None,
+                 fp32_model_path=None,
+                 cpu_math_library_num_threads=1):
+        """
+        Args:
+            int8_model_save_path(str): The path to save model for MKL-DNN int8 inference.
+                            None means it doesn't save model. defalut: None.
+            fp32_model_path(str): The path of fp32 model to be converted to int8 model
+                            None means it doesn't have fp32 model. defalut: None.
+            cpu_math_library_num_threads(int): The num of omp threads user want 
+                            MKLDNNPostTrainingQuantStrategy to use. defalut: 1.
```

Comment:
defalut->default. @zhaify @bingyanghuang Please check all the notes again, thanks very much!